tests/ops/op_math/test_controlled.py

```diff
@@ -1440,7 +1418,7 @@ def test_multi_ctrl():
     op = tape.operations[0]
     assert isinstance(op, Controlled)
     new_tape = tape.expand(depth=1)
-    assert qml.equal(new_tape[0], qml.Toffoli(wires=[3, 7, 0]))
+    assert equal_list(new_tape[0], qml.MultiControlledX(wires=[3, 7, 0]))
```

Comment:
```suggestion
    assert qml.equal(new_tape[0], qml.MultiControlledX(wires=[3, 7, 0]))
```

We are simply comparing too operations.  No reason to use a helper function for that, when `qml.equal` is familiar and gets the job done.